src/web/web-service/wpan_service.hpp

```diff
@@ -146,7 +148,7 @@ class WpanService
      * @param[in]  aIfName  The pointer to the interface name of wpantund.
      *
      */
-    void SetInterfaceName(const char *aIfName) { strncpy(mIfName, aIfName, sizeof(mIfName)); }
+    void SetInterfaceName(const char *aIfName) { strcpy_safe(mIfName, sizeof(mIfName), aIfName); }
```

Comment:
`strcpy_safe` will truncate the string to `""` if the argument is too long. It is different to previous impl, please make sure this is the expected behavior.

Reply:
I admit this may not be perfect. But I think `strcpy_safe` is better than the current `strncpy`. We can further enhance this part in separate PRs. Thoughts?

Reply:
ok to me.